File: 백준/Bronze/10810. 공 넣기/공 넣기.java
import java.io.*;

class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] str = br.readLine().split(" ");
        int n = Integer.parseInt(str[0]);
        int m = Integer.parseInt(str[1]);
        
        int[] arr = new int[n];
        
        for(int i = 0 ; i < m ; i++){
            String[] input = br.readLine().split(" ");
            int k = Integer.parseInt(input[2]);
            for(int z = Integer.parseInt(input[0]) - 1 ; z < Integer.parseInt(input[1]); z++){
                arr[z] = k;
            }
        }
        
        for(int r : arr){
            System.out.print(r + " ");
        }
    }    

}

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys

n,m = map(int,input().split())
basket = [0 for i in range(n)]

for _ in range(m):
  i,j,k = map(int, sys.stdin.readline().rstrip().split())
  for z in range(i-1, j):
    basket[z] = k

for i in basket:
    print(i, end=" ")